sql/src/test/java/org/opensearch/sql/sql/SQLServiceTest.java

```diff
@@ -83,6 +84,115 @@ public void onFailure(Exception e) {
         });
   }
 
+  @Test
+  public void canExecuteJsonFormatRequest() {
+    doAnswer(invocation -> {
+      ResponseListener<QueryResponse> listener = invocation.getArgument(1);
+      listener.onResponse(new QueryResponse(schema, Collections.emptyList(), ""));
+      return null;
+    }).when(queryService).execute(any(), any());
+
+    sqlService.execute(
+        new SQLQueryRequest(new JSONObject(), "SELECT FIELD FROM TABLE", QUERY, "json"),
+        new ResponseListener<QueryResponse>() {
+          @Override
+          public void onResponse(QueryResponse response) {
+            assertNotNull(response);
+          }
+
+          @Override
+          public void onFailure(Exception e) {
+            fail(e);
+          }
+        });
+  }
+
+  @Test
+  public void canThrowUnsupportedExceptionForAggregationJsonQuery() {
```

Comment:
Could you replace `can` with `test` in test method names?

`can...` is not one of the common unit test naming convention.